doc/modules/ensemble.rst

```diff
@@ -1031,22 +1031,23 @@ Vector Machine, a Decision Tree, and a K-nearest neighbor classifier::
    >>> from sklearn.svm import SVC
    >>> from itertools import product
    >>> from sklearn.ensemble import VotingClassifier
-
+   >>>
```

Comment:
Why do you add these ``>>>``s (here and below)

Reply:
In python console we can't create empty string:
```
d.kataev:scikit-learn§ python
Python 2.7.15 (default, Jun 17 2018, 12:46:58)
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
Could not open PYTHONSTARTUP
IOError: [Errno 2] No such file or directory: 'bootstrap.py'
>>> a = 1
>>>
>>>
>>> b = 2
```

Reply:
``>>>`` is a symbol for doctests, and extra ``>>>`` makes the example a bit strange. We have some blank lines in other places and I think it's acceptable.
See e.g., https://35474-843222-gh.circle-artifacts.com/0/doc/modules/ensemble.html#weighted-average-probabilities-soft-voting, you can find that extra ``>>>`` is unfriendly :)

Reply:
Remove empty console lines.